static/src/javascripts/projects/common/views/commercial/creatives/gu-style-comcontent.html

```diff
@@ -1,13 +1,19 @@
-<a class="<%=data.gustyleClass%>" href="<%=data.clickMacro%><%=data.articleUrl%>" data-link-name="creative | gu style commercial content">
-    <div class="u-responsive-ratio inlined-image">
+<div class="gu-display" data-link-name="creative | gu style commercial display content">
+    <div class="gu-image-layer u-responsive-ratio inlined-image">
         <img class="gu-image" src="<%=data.articleImage%>">
     </div>
-    <div class="gu-box">
-        <h2 class="gu-title"><%=data.articleHeaderText%></h2>
-        <%=data.standFirst%>
-        <div class="gu-brand">
-        	<%=data.noteOrLink%>
-        	<img class="gu-logo" src="<%=data.brandLogo%>" alt="<%=data.brand%>">
-        </div>
+    <div class="gu-mutable gu-content">
+        <a href="<%=data.clickMacro%><%=data.articleUrl%>" class="gu-link" data-link-name="header text">
+            <h2 class="gu-content__title"><%=data.articleHeaderText%></h2>
+        </a>
+        <a href="<%=data.clickMacro%><%=data.articleUrl%>" class="gu-link" data-link-name="text">
```

Comment:
It's not necessary. But with one `A` when you will hover over a title, the whole text will be underlined. It looks ugly.